src/wet-lab/design/solid-phase.md

```diff
@@ -1 +1,51 @@
 # Solid phase DNA Synthesis
+
+## Solid phase DNA Synthesis
+
+This document outlines the process of non-traditional solid phase DNA synthesis through the use of tdt. In particular, this document will provide a broad overview of the workflow and the details of primer immobilization, solid phase chemistry, and primer cleavage. It also outlines the chemistry of monomers intended to use.
+Context and Scope
+Traditional solid phase chemical synthesis occurs from 3’ -> 5’ as shown below.
+![chemical solid phase DNA synthesis](Chemical-SPPS.png)
```

Comment:
For images, add a new line before and after the image so the image is on it's own line and there isn't text beside the image.

Would look like this: 
![image](https://github.com/UBC-iGEM/internal-wiki-2023-24/assets/55033656/906c9206-ac40-4f8f-a679-a1bcd93eaa2d)

Without the new lines:
![image](https://github.com/UBC-iGEM/internal-wiki-2023-24/assets/55033656/3b5cd8d5-870b-4cbb-8876-4283843cc424)

With new lines:
![image](https://github.com/UBC-iGEM/internal-wiki-2023-24/assets/55033656/13f29a60-5784-4f6f-a95b-f8fd3175c031)

If this could be applied to all images that would be great!